packages/db/src/translate/providers.ts

```diff
@@ -41,6 +41,9 @@ export const translateApiProviderToPostgresProvider = async (
     port: record.port,
     host: record.host,
     protocol: record.protocol,
+    allowed_redirects: record.allowedRedirects
+      ? JSON.stringify(record.allowedRedirects)
```

Comment:
See above. Storing this as an array would save steps like this but 🤷

Reply:
Resolved here: https://github.com/nasa/cumulus/commit/b3a173b7afaf8df2689665f914d2e14bcb0355fd